### DISCUSSION TITLE
Ldap starttls

### DISCUSSION THREAD

**Discussion by ibrahimdjadir** [authelia/authelia#5582] [Q&A] [answered]
Inorder to configure ldap authentication back end I did the following config 
` ldap:
     implementation: custom
     url: ldap://ldap.my_domaine:389
     timeout: 5s
     start_tls: true
     tls:
       server_name: my.ldap.server
       skip_verify: false
       minimum_version: TLS1.2
       maximum_version: TLS1.3`
I am using self signed certf for my server so I added myldap ldap CA cert to certificates_directory
when I run my container I get the folloing erro : level=error msg="Failure running the user provider startup check: starttls failed with error: LDAP Result Code 200 \"Network Error\": TLS handshake failed (x509: certificate relies on legacy Common Name field, use SANs instead)"

**Comment by james-d-elliott** [comment, marked as answer]
Common Name is long deprecated (in 2000 by [RFC2818](https://datatracker.ietf.org/doc/html/rfc2818#section-3.1)), Following this and several security issues later, several programming languages including go decided to stop backwards comparability for this long deprecated functionality.

That's what  the error is telling you. Here's a guide I found randomly: https://raymii.org/s/tutorials/OpenSSL_generate_self_signed_cert_with_Subject_Alternative_name_oneliner.html
